rmgpy/rmg/reactors.py

```diff
@@ -35,25 +35,12 @@
 import logging
 import itertools
 
-if __debug__:
-    try:
-        from os.path import dirname, abspath, join, exists
-
-        path_rms = dirname(dirname(dirname(abspath(__file__))))
-        from julia.api import Julia
-
-        jl = Julia(sysimage=join(path_rms, "rms.so")) if exists(join(path_rms, "rms.so")) else Julia(compiled_modules=False)
```

Comment:
Are we breaking workflows for people that use system images to avoid compilation?

Reply:
Do you mean by using a (global) Julia binary instead of the one come with the conda environment?

Reply:
I don't think any of us is using this function, and I have also never used this. I don't think we should get ahead of ourselves to provide a service that maybe no one will use. I prefer to wait till someone complains about not having this option anymore, or just let them figure it out rather than doing their work for them.

Reply:
+1